libcontainer/devices/devices_unix.go

```diff
@@ -94,6 +94,9 @@ func getDevices(path string) ([]*configs.Device, error) {
 			if err == ErrNotADevice {
 				continue
 			}
+			if os.IsNotExist(err) {
```

Comment:
@rhatdan, If you return error if device is not present, not sure what will caller do.

I think this whole function is racy. Caller passes in a  list of devices, and caller has not opened the devices in there. That means devices can disappear any time.

I think we should put a comment that this function goes through a list of devices and if some devices are not present, it ignores them.

Returning error if some device is not present, will not help.